app/classes/query/modules/validation.rb

```diff
@@ -27,7 +27,11 @@ def check_for_unexpected_params(old_params)
 
   def validate_value(param_type, param, val)
     if param_type.is_a?(Array)
-      array_validate(param, val, param_type.first).flatten
+      result = array_validate(param, val, param_type.first).flatten
```

Comment:
Is `result` ever an array of numerical strings, e.g., `["123", "145", ...]`?
If yes, should it be handled the same as an array of integers?